test/mock/createSearchClient.js

```diff
@@ -0,0 +1,26 @@
+import {
```

Comment:
The current Jest configuration doesn't support TypeScript files from this folder.

Reply:
Handled in #3132.